docs/INSTALL.md

```diff
@@ -101,6 +101,36 @@ e.g.: `make LLVM_CONFIG=llvm-config-14`
 
 macOS has some gotchas due to the idiosyncrasies of the platform.
 
+macOS supports SYSV shared memory used by AFL's instrumentation, but the
+default settings aren't usable with AFL++. The default settings (on 10.14 through
+14.2 at least) seem to be:
+
+```shell
+$ ipcs -M
+IPC status from <running system> as of XXX
+shminfo:
+        shmmax: 4194304 (max shared memory segment size)
+        shmmin:       1 (min shared memory segment size)
+        shmmni:      32 (max number of shared memory identifiers)
+        shmseg:       8 (max shared memory segments per process)
+        shmall:    1024 (max amount of shared memory in pages)
+```
+
+To temporarily change your settings to something usable with AFL++,
+run these commands:
+
```

Comment:
Wouldn’t it be better to recommend to execute afl-system-config instead?

Reply:
Yeah, looks like!

A shame that INSTALL.md does not mention it at all, I didn't know it existed.

Will update PR...

Reply:
Nothing is perfect what we should strive for perfection :)

Reply:
PR now updated.